applications/cosmoly.md

```diff
@@ -0,0 +1,127 @@
+- **Nexo Earn API**: Off-chain earning services with 5-16% APY (only with some tokens and stable coins)
+
+#### Internal services
+
+- **Bridge between VeChain and Polygon (and 22 blockchains more):** This would allow VeChain to connect (using polygon) to the more than 22 different blockchains of the Li.Fi protocol. Without the need to go through an CEX, registrations or verifications.
+- **Earn protocolo in VeChain:** Earn daily interest using VeUSD, VET and Cosmo-USD (a token collateralized in USDC/USDT).
+- **Creation of COSMO-USD:** A token collateralized in USDC/USDT on the polygon blockchain, which, using the Li.Fi protocol, connects to more stablecoins.
+
+#### Benefits of working with VeChain Services
+
+- Great overall support from the Vechain team
+- Increase in user adoption for Vechain tokens and DeFi ecosystem
+- Lower Gas fee
+- Faster transaction confirmation times
+
+#### Documentation
+
+You can check our Litepaper at: <https://docs.cosmolylabs.com>
+
+#### MVP
+
+We are a team focused more on action, you can check our platform at <https://cosmolylabs.com>
+
+### Ecosystem Fit
+
+Here is our competitive advantage, there is no project of this category on VeChain. The only closest competitors would be the CEX, which are in another market category.
+
+And according to <https://apps.vechain.org/#defi> the only application with a bridge function is <https://swap.vechain.energy/> but it does not have savings functions
+
+Our idea is to connect Vechain's currently operating DEXs, to increase liquidity, obtain the best rates and improve the user experience
+
+## Team
+
+### Team members
+
+- David Gimenez (Full Stack Developer | Blockchain Developer | Graphic Designer UI/UX | CEO CosmolyLabs)
+
+### Team Website
+
+- <https://cosmolylabs.com>
+
+### Team's experience
+
+I am David Gimenez, an experienced software developer and entrepreneur with a special focus on blockchain, finance and UI/UX design. I have a strong background in TypeScript, React.js, Angular.js, Flutter, Python, MongoDB, SQL, Solidity, Web3 and PyTeal.
+
+I have a passion for creating efficient, high-quality and user-friendly applications which are both secure and reliable. I strive to stay up to date with the latest technologies and industry trends by attending conferences, workshops and online courses.
+
+Throughout my career, I have worked on a wide variety of projects, from complex multi-node architectures to user experience design. I have a strong eye for detail, and I am always ready to take on challenges.
+
+You can check my LinkedIn to know more about me.
+
+For smaller tasks (marketing, smart contract audits, graphic design, UX/UI creation for Vechain integration) external agents would be hired (using Fiverr or Upwork)
+
+### Team Code Repos
+
+- <https://github.com/AgentDavid>
+
+### Team LinkedIn Profiles
+
+- <https://www.linkedin.com/in/davidgx100/>
+
+## Development Roadmap
+
+|  | Milestone 1 | Milestone 2 | Total |
```

Comment:
https://github.com/vechain/grant-program/blob/master/applications/application-template.md#overview-1 We have a rule that limits milestone 1 funds should not be higher than 40% of the total amount you are applying for, I would suggest splitting milestone1 into 2 milestones, thank you in advance!